pkgs/by-name/sp/spacedrive/package.nix

```diff
@@ -63,15 +63,16 @@ else appimageTools.wrapType2 {
   extraInstallCommands =
     let
       appimageContents = appimageTools.extractType2 { inherit pname version src; };
+      desktopFile = "com.${pname}.desktop";
     in
     ''
       # Remove version from entrypoint
-      mv $out/bin/spacedrive-"${version}" $out/bin/spacedrive
+      mv $out/bin/${pname}-${version} $out/bin/${pname}
```

Comment:
I'm not sure these changes should be applied. See, https://github.com/NixOS/nixpkgs/pull/270121#discussion_r1405432291